src/integration-test/java/org/ow2/proactive/catalog/rest/controller/BucketControllerIntegrationTest.java

```diff
@@ -109,127 +105,64 @@ public void testCreateDuplicatedBucketSameUser() {
     }
 
     @Test
-    public void testCreateDuplicatedBucketDifferentUsers() {
```

Comment:
duplicated test with testListOneBucketNameTwoOwners(), in this latter test, it includes already the duplicated bucket with different users